apollo-router/feature_discussions.json

```diff
@@ -1,6 +1,8 @@
 {
+  "experimental": {
     "experimental_retry": "https://github.com/apollographql/router/discussions/2241",
     "experimental_response_trace_id": "https://github.com/apollographql/router/discussions/2147",
-    "experimental_logging": "https://github.com/apollographql/router/discussions/1961",
-    "experimental_jwt_authentication": "https://github.com/apollographql/router/discussions/2391"
```

Comment:
Was it intentional to remove JWT from this?

Reply:
Yes, the feature itself is no longer experimental.  This should have been removed already, but just hadn't been.

Reply:
It sounds like yes, since JWT auth is no longer experimental.